tasmota/tasmota_template_ESP32.h

```diff
@@ -44,7 +44,7 @@
 
 enum UserSelectablePins {
   GPIO_NONE,                           // Not used
-  GPIO_KEY1, GPIO_KEY1_NP, GPIO_KEY1_INV, GPIO_KEY1_INV_NP,  // 4 x Button
+  GPIO_KEY1, GPIO_KEY1_NP, GPIO_KEY1_INV, GPIO_KEY1_INV_NP, GPIO_KEY1_TC, // 4 x Button + Touch
```

Comment:
Inserting anything here will break current gpio configs. It has to be inserted at the end of the list.